packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -164,6 +166,17 @@ storiesOf('Components|Paragraph', module)
 
 The above example dismisses the use of the `addDecorator` method and decorates the story directly.
 
+### buildRTLSubstories
+
+```jsx
+// storybook/config.js
+
+import { buildRTLSubstories } from '@bbc/psammead-storybook-helpers';
+
+// must be placed after the storybook configure function
+buildRTLSubstories();
```

Comment:
So if I understood well, just adding this in the `.storybook/config.js`, will create the RTL variants for each story?

Reply:
Correct - https://github.com/bbc/psammead/pull/2419/files